app/controllers/users_controller.rb

```diff
@@ -1,4 +1,26 @@
 class UsersController < ApplicationController
   def new
+  	@user = User.new
   end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
1. What will happen if it don't find the user with params[:id] 
2. Can you study about find and find_by

Reply:
If it don't find the user with params[:id] , It will raises an Exception : ActiveRecord::RecordNotFound and server is crashed
find_by will return nil in this case.
I answered above